src/components/parameters/Parameters.svelte

```diff
@@ -24,51 +25,74 @@
   $: sortedFormParameters = formParameters.sort((a, b) => compare(a.order, b.order));
 </script>
 
-{#each sortedFormParameters as formParameter (formParameter.name)}
-  <Highlight highlight={highlightKeysMap[formParameter.name]}>
-    <div bind:clientWidth class="parameter">
-      {#if formParameter.schema.type === 'series' || formParameter.schema.type === 'struct'}
-        <ParameterRec
-          {disabled}
-          {expanded}
-          {formParameter}
-          {hideRightAdornments}
-          {labelColumnWidth}
-          {level}
-          {levelPadding}
-          {parameterType}
-          on:change
-          on:reset
-          {use}
-        />
-      {:else}
-        <ParameterBase
-          {disabled}
-          {formParameter}
-          {hideRightAdornments}
-          {labelColumnWidth}
-          {level}
-          {levelPadding}
-          {parameterType}
-          on:change
-          on:reset
-          {use}
-        />
-      {/if}
-    </div>
-  </Highlight>
-{/each}
+<div class="parameters-container">
+  {#each sortedFormParameters as formParameter (formParameter.name)}
+    <Highlight highlight={highlightKeysMap[formParameter.name]}>
+      <div bind:clientWidth class="parameter">
+        {#if formParameter.schema.type === 'series' || formParameter.schema.type === 'struct'}
+          <ParameterRec
+            {disabled}
+            {expanded}
+            {formParameter}
+            {hideRightAdornments}
+            {labelColumnWidth}
```

Comment:
Probably not the time or the place but at some point should we try to make these inputs longer? There's definitely a risk of misreading input values given how many things are now taking up space on the right side plus the inputs are pretty small to begin with. 

Ex of misleading input:
<img width="409" alt="image" src="https://github.com/NASA-AMMOS/aerie-ui/assets/4419607/98b706b4-992d-418f-b429-c6f07fd3f01f">

When you make the panel larger you can see the remainder of the number:
<img width="532" alt="image" src="https://github.com/NASA-AMMOS/aerie-ui/assets/4419607/e13ef864-2f6a-494f-80df-5d775fd9f8ce">